packages/user-profile-spa/src/app/user-profile-components/header/header.component.html

```diff
@@ -0,0 +1,10 @@
+<!-- Header component starts -->
+<header>
+  <div class="breadcrumb">
+    <a href="#" class="active">Home</a>
+    <i class="icon ion-ios-arrow-forward"></i>
+    <a href="#">My Profile</a>
+  </div>
+  <h4 class="app-title">My Profile</h4>
+</header>
+<!-- Header component ends -->
```

Comment:
@hybridx @gisumit @deshmukhmayur Shouldn't this go under the SSI service?

Reply:
@sayak-sarkar No, it's not common in all the apps. It will be converted into a web component in the component library later.\
So any apps that need it can import and use it.

Reply:
Ok. Makes sense!